app/emails/email.worker.server.ts

```diff
@@ -0,0 +1,59 @@
+import { transporter } from "~/emails/transporter.server";
+import { QueueNames, scheduler } from "~/utils/scheduler.server";
+import type { EmailPayloadType } from "./types";
+import { SMTP_FROM } from "../utils/env";
+import { ShelfError } from "../utils/error";
+
+export const registerEmailWorkers = async () => {
+  await scheduler.work<EmailPayloadType>(
+    QueueNames.emailQueue,
+    { newJobCheckIntervalSeconds: 60 * 3, teamSize: 2 },
```

Comment:
Can you please add a comment on how this relates to our limit per second of sending. Maybe just place the whole calculation we discussed on discord so we have it in the code for future reference.